bin/oneoff/generate_legacy_survey_summaries.rb

```diff
@@ -150,7 +150,7 @@ def snapshot_summer_workshops_from_jotform(course)
   end
 
   # delete all existing
-  Pd::LegacySurveySummary.where(facilitator_id: nil).where(course: course, subject: subject).delete_all
+  Pd::LegacySurveySummary.where(course: course, subject: subject).delete_all
```

Comment:
could we get into issues because the default above is to only run for the first 10 facilitators? If we accidentally run this without changing that it will wipe out almost everyone's data

Reply:
That's a fair point, and I've fixed it.  I've been checking the data's before & after count for each category after running the script as a way of sanity-checking each run.